_episodes/03-develop-env.md

```diff
@@ -157,77 +208,97 @@ The Lmod tool will load these dependencies for you when you load the module (see
+ 3) cray-mpich/8.0.10(default)       7) libfabric/1.10.0.0.249(default)  11) xpmem/2.2.35-7.0.1.0_3.9__gfa8d091.shasta(default)  
+ 4) PrgEnv-cray/7.0.0(default)       8) craype-network-slingshot10 
 ```
 {: .output}
 
-You load modules with the `module load` (or `ml`) command. For example, to load the `netcdf` module:
+You load modules with the `module load` command. For example, to load the `cray-netcdf` module:
 
 ```
-[auser@archer2-login1 ~]$ ml netcdf
+auser@login01-nmn:~> module load cray-netcdf
 ```
-{: .bash}
+<!-- Commented out until Lmod is really available and we can test{: .language-bash}
 
 > ## `ml` has two meanings
 > Notice the the convenience command `ml` has a different meaning depending on whether or not you 
 > supply an argument. Without an argument it means `module list` and with an argument, it means
 > `module load`.
-{: .callout
-
-}
+{: .callout} -->
 
-Now, lets list our loaded modules again with `ml`:
+Now, lets list our loaded modules again with `module list`:
 
 ```
-[auser@archer2-login1 ~]$ ml
+auser@login01-nmn:~> module list
 ```
-{: .bash}
+{: .language-bash}
 ```
-Currently Loaded Modules:
-  1) autotools   3) intel/19.0.3.199   5) ohpc                 7) phdf5/1.10.4
-  2) prun/1.3    4) impi/2019.3.199    6) packages-nextgenio   8) netcdf/4.6.2
+Currently Loaded Modulefiles:
+ 1) cce/10.0.0(default)              5) craype/2.6.4(default)             9) cray-dsmml/0.1.0(default)                           
+ 2) cray-libsci/20.03.1.4(default)   6) craype-x86-rome                  10) perftools-base/20.05.0(default)                     
+ 3) cray-mpich/8.0.10(default)       7) libfabric/1.10.0.0.249(default)  11) xpmem/2.2.35-7.0.1.0_3.9__gfa8d091.shasta(default)  
+ 4) PrgEnv-cray/7.0.0(default)       8) craype-network-slingshot10       12) cray-netcdf/4.6.3.2(default) 
 ```
 {: .output}
 
-You can see that as well as the default `netcdf` module (`netcdf/4.6.2` as we did not specify a version
-explicitly) Lmod has also loaded the dependencies for the default `netcdf` module.
+You can see that as well as the default `cray-netcdf` module (`cray-netcdf/4.6.3.2` as we did not specify a version
+explicitly).
+
+<!--
+  Note: Once Lmod is working, we likely want to choose a module that has dependencies so we 
+  can note that Lmod has automatically loaded dependencies. -->
 
 If you want to swap two versions of the same module then you simply load the version that you 
 want to swap for the currently loaded version, Lmod recognises that they are the same module 
-with different versions and swaps them for you. 
+with different versions and swaps them for you.
 
 ## Switching between compiler environments
 
-You use the `module swap` command to switch between the three different compiler envionments available
+You use the `module swap` command to switch between the three different compiler environments available
 on ARCHER2. The available environments are:
 
```

Comment:
two different compilers now until the AMD ones are available

Reply:
Thanks for spotting that. I've updated the number to "two".